definitions/divorce/json/CaseTypeTab/CaseTypeTab-nonprod.json

```diff
@@ -0,0 +1,79 @@
+[
+    {
+        "LiveFrom": "01/01/2017",
+        "CaseTypeID": "DIVORCE",
+        "Channel": "CaseWorker",
+        "TabID": "coRespondent",
+        "TabLabel": "Co-Respondent",
+        "TabDisplayOrder": 22,
+        "CaseFieldID": "CoRespondentSolicitorRepresented",
+        "TabFieldDisplayOrder": 26,
+        "FieldShowCondition": "CoRespondentSolicitorRepresented=\"Y*\""
+    },
+    {
+        "LiveFrom": "01/01/2017",
+        "CaseTypeID": "DIVORCE",
+        "Channel": "CaseWorker",
+        "TabID": "coRespondent",
+        "TabLabel": "Co-Respondent",
+        "TabDisplayOrder": 22,
+        "CaseFieldID": "CoRespondentSolicitorName",
+        "TabFieldDisplayOrder": 27,
+        "FieldShowCondition": "CoRespondentSolicitorRepresented=\"Y*\""
+    },
+    {
+        "LiveFrom": "01/01/2017",
+        "CaseTypeID": "DIVORCE",
+        "Channel": "CaseWorker",
+        "TabID": "coRespondent",
+        "TabLabel": "Co-Respondent",
+        "TabDisplayOrder": 22,
+        "CaseFieldID": "CoRespondentSolicitorAddress",
+        "TabFieldDisplayOrder": 28,
+        "FieldShowCondition": "CoRespondentSolicitorRepresented=\"Y*\""
+    },
+    {
+        "LiveFrom": "01/01/2017",
+        "CaseTypeID": "DIVORCE",
+        "Channel": "CaseWorker",
+        "TabID": "coRespondent",
+        "TabLabel": "Co-Respondent",
+        "TabDisplayOrder": 22,
+        "CaseFieldID": "CoRespondentSolicitorPhone",
+        "TabFieldDisplayOrder": 29,
+        "FieldShowCondition": "CoRespondentSolicitorRepresented=\"Y*\""
+    },
+    {
+        "LiveFrom": "01/01/2017",
+        "CaseTypeID": "DIVORCE",
+        "Channel": "CaseWorker",
+        "TabID": "coRespondent",
+        "TabLabel": "Co-Respondent",
+        "TabDisplayOrder": 22,
+        "CaseFieldID": "CoRespondentSolicitorEmail",
+        "TabFieldDisplayOrder": 30,
+        "FieldShowCondition": "CoRespondentSolicitorRepresented=\"Y*\""
+    },
+    {
+        "LiveFrom": "01/01/2017",
+        "CaseTypeID": "DIVORCE",
+        "Channel": "CaseWorker",
+        "TabID": "coRespondent",
+        "TabLabel": "Co-Respondent",
+        "TabDisplayOrder": 22,
+        "CaseFieldID": "DerivedCoRespondentSolicitorAddr",
+        "TabFieldDisplayOrder": 31,
+        "FieldShowCondition": "CoRespondentSolicitorRepresented=\"Y*\""
+    },
+    {
+        "LiveFrom": "01/01/2017",
+        "CaseTypeID": "DIVORCE",
+        "Channel": "CaseWorker",
+        "TabID": "coRespondent",
+        "TabLabel": "Co-Respondent",
+        "TabDisplayOrder": 22,
+        "CaseFieldID": "CoRespSolAOSSOT",
+        "TabFieldDisplayOrder": 32,
+        "FieldShowCondition": "CoRespondentSolicitorRepresented=\"Y*\""
```

Comment:
do we set it to YES and Yes?

Reply:
I'm just covering both scenarios. Just in case.
We set it to "Yes"